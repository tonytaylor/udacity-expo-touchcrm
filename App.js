import { Provider } from "react-redux";

import AppNavigation from './src/navigation';

import initializeStore from "./src/store";

const store = initializeStore();

export default function App() {
  return (
    <Provider store={store}>
      <AppNavigation />
    </Provider>
  );
}
