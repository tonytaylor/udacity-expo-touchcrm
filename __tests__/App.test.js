import React from 'react';
import renderer from 'react-test-renderer';


import App from '../App';

describe('<App />', () => {
  it('has two children', () => {
    const tree = renderer.create(<App />).toJSON();

    expect(tree.children.length).toBe(1);
  });
  it('renders w/o issue', () => {
    const tree = renderer.create(<App />).toJSON();
    expect(tree).toMatchSnapshot();
  })
});