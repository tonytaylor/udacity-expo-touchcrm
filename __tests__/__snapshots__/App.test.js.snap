// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> renders w/o issue 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text>
    Open up App.js to start working on your app!
  </Text>
  <Text>
    Does this break the build? (yay)
  </Text>
</View>
`;
